File: MasterHacker.py
import random

while True:
    words = ["the blockchain", "AI", "machine learning", "neural net", "DNS server", "an API", "coding and algorithms"]
    choice = input("Would you like predetermined words? (Y/N): ")
    if choice.lower() == "n" or choice.lower() == "no" or choice.lower() == "y" or choice.lower() == "yes":
        random.shuffle(words)
        if choice.lower() == "n" or choice.lower() == "no":
            for i in range (0, len(words)):
                words[i] = input("Please enter a word (" + str(i+1) + "/7): ")
        print("Me, was pretty easy. Used " + words[0] + " to occupy space on the Reddit network."
              + " Once traffic came through for that account I flipped the " + words[1] + " to my PC"
              + " and harvested the password as it was being transmitted. I also employed " + words[2]
              + " to gain " + words[3] + " access on their network."
              + " From there it was a simple job of breaking through " + words[4] + "s and getting"
              + " into local " + words[5] + " accounts."
              + " Grabbed a few passwords and started hacking with " + words[6] + ".")
    else:
        print("Invalid option.")

File: MasterHacker.java
import java.util.ArrayList;

public class MasterHacker
{
	public static void main(String[] args)
	{
		String[] comment = {"Me, was pretty easy. Used a reverse ", " to occupy space on the Reddit ", ". Once traffic came through for that account I flipped the receiving ", " to my ", " and harvested the password as it were being transmitted. I also opened a ", " to gain ", " access on their ", ". From there it was a simple job of breaking through the ", " and getting into the ", ". Grabbed a few passwords and started hacking the accounts."};
		String[] techwords = {"DNS transcoder", "network", "address", "PC", "backwards tunnel", "LAN", "network", "firewalls", "local admin accounts", "AI", "blockchain", "neural network"};
		
		techwords = shuffle(techwords);
		
		System.out.println(comment[0]);
		for (int i = 0; i < comment.length - 1; i++)
		{
			System.out.print(techwords[i] + comment[i + 1]);
		}
	}
	
	public static String[] shuffle(String[] in) //shuffle method because Java doesnt have one
	{
		String[] out = new String[in.length];
		ArrayList<String> inA = new ArrayList<String>();
		for (int i = 0; i < in.length; i++)
		{
			inA.add(in[i]);
		}
		
		for (int i = 0; i < in.length; i++)
		{
			int remove = (int) (Math.random() * inA.size());
			out[i] = inA.get(remove);
			inA.remove(remove);
		}
		return out;
	}
}
